el_client.el

```diff
@@ -63,6 +63,8 @@
 
 (defvar xelb-request-fields nil "Fields in the current request.")
 
+(defconst xelb-xproto-namespace "xproto:" "The namespace of the core protocol.")
```

Comment:
The usage of constants does not seem coherent in el_client.el. I've seen there is a variable xelb-prefix, but also many "xcb:" literal strings. But it seems to be the right thing to extract such constants like you have done here.